lib/eslint.js

```diff
@@ -863,11 +863,13 @@ module.exports = (function() {
      * @param {string} message The actual message.
      * @param {Object} opts Optional template data which produces a formatted message
      *     with symbols being replaced by this object's values.
+     * @param {Object} fix A fix command description.
```

Comment:
I'm a bit confused. Based on documentation above, `fix` is a `function` for fixing current issue, right? If that's correct, then this should be a bit more clear.